phpunit.xml.dist

```diff
@@ -1,26 +1,29 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <phpunit
-        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd"
-        backupGlobals="false"
-        colors="true"
-        bootstrap="tests/bootstrap.php"
-        failOnRisky="true"
-        failOnWarning="true">
-  <coverage>
-    <include>
-      <directory>./src</directory>
-    </include>
-  </coverage>
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
+    bootstrap="tests/bootstrap.php"
+    cacheResultFile=".phpunit.cache/test-results"
+    colors="true"
+    failOnRisky="true"
+    failOnWarning="true"
+>
+    <php>
+        <ini name="display_startup_errors" value="On"/>
+        <ini name="display_errors" value="On"/>
+        <ini name="error_reporting" value="-1"/>
+        <env name="SYMFONY_DEPRECATIONS_HELPER" value="disabled"/>
```

Comment:
We don't use the [symfony/phpunit-bridge](https://github.com/symfony/phpunit-bridge), do we?